### DISCUSSION TITLE
Intall the mime type .qmd in albert launcher

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#3935] [Q&A] [answered]
I would like [albert](https://github.com/albertlauncher/albert) to index my .qmd files.

I already opened an [issue](https://github.com/albertlauncher/plugins/issues/85) on albert and the developer told me that quarto has to install the mime type on my system, so I am asking here.

**Comment by mcanouil** [comment]
You can do that yourself, although the method might depends on your system, for example, <https://askubuntu.com/questions/586476/how-to-assign-set-a-mime-type-to-a-file>.

**Comment by ghost** [comment, marked as answer]
Albert now supports .qmd files.
